lib/samples/create_buffers_around_points/create_buffers_around_points.dart

```diff
@@ -0,0 +1,446 @@
+  final tapPointGraphicsOverlay = GraphicsOverlay();
+
+  // Fill symbols for buffer and tap points.
+  final bufferFillSymbol = SimpleFillSymbol();
+  final tapPointSymbol = SimpleMarkerSymbol();
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+        top: true,
+        child: Stack(
+          children: [
+            Column(
+              children: [
+                Expanded(
+                  // Add a map view to the widget tree and set a controller.
+                  child: ArcGISMapView(
+                    controllerProvider: () => _mapViewController,
+                    onMapViewReady: onMapViewReady,
+                    onTap: onTap,
+                  ),
+                ),
+                Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                  children: [
+                    ElevatedButton(
+                      onPressed: () {
+                        showModalBottomSheet(
+                          context: context,
+                          builder: (BuildContext context) {
+                            return StatefulBuilder(
+                              builder: (BuildContext context, StateSetter setState) {
+                                return buildSettings(context, setState);
+                              },
+                            );
+                          },
+                        );
+                      },
+                      child: const Text('Union Settings'),
+                    ),
+
+                    // A button to clear the buffers.
+                    ElevatedButton(
+                      onPressed: _bufferPoints.isEmpty
+                          ? null
+                          : () {
+                        setState(() {
+                          clearBufferPoints();
+                          _status = Status.addPoints;
+                        });
+                      },
+                      child: const Text('Clear'),
+                    ),
+                  ],
+                ),
+              ],
+            ),
+            // Display a progress indicator and prevent interaction until state is ready.
+            Visibility(
+              visible: !_ready,
+              child: const SizedBox.expand(
+                child: ColoredBox(
+                  color: Colors.white30,
+                  child: Center(child: CircularProgressIndicator()),
+                ),
+              ),
+            ),
+            Positioned(
+              top: 0,
+              left: 0,
+              right: 0,
+              child: Container(
+                padding: const EdgeInsets.all(8),
+                color: Colors.white.withOpacity(0.4),
+                child: Text(
+                  _status.label,
+                  textAlign: TextAlign.center,
+                ),
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+
+
```

Comment:
there should just be one blank line here instead of 2

Reply:
Removed the additional new line at bottom.